CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 To publish a new release run `scripts/release` from the project directory.
 
 ## [Unreleased]
+### Fixed
+- Fixed resizing the image in [kitty] (see [GH-124])
```

Comment:
Can you also link this PR, i.e. GH-133?

Reply:
And add you name (see below in the changelog, where kitty got added), as dedication and thanks ? :hugs: